src/components/EmptyState/EmptyState.stories.mdx

```diff
@@ -33,32 +47,34 @@ export const doNothing = () => {};
 
 <Canvas>
   <Story name="With content">
-    <EmptyState
-      image={
-        <Icon
-          name="containers"
-          style={{
-            opacity: 0.25,
-            height: "2.5rem",
-            width: "2.5rem",
-            marginBottom: "1rem"
-          }}
-        />
-      }
-      title="No instances found"
-    >
-      <p>
-        There are no instances in this project. Spin up your first instance!
-      </p>
-      <p>
-        <a href="#todo" target="_blank">
-          How to create instances
-          <Icon name={ICONS.externalLink}/>
-        </a>
-      </p>
-      <Button appearance="positive" onClick={doNothing}>
-        Create instance
-      </Button>
-    </EmptyState>
+    {() => (
```

Comment:
Why story contents need to be returned from function rather then being direct children?

Reply:
Temporary workaround for mdx new lines we discussed before (extra paragraph wrapper). https://github.com/canonical/react-components/issues/1001